Dockerfile

```diff
@@ -31,23 +33,24 @@ RUN set -xe; \
     mix deps.compile --all; \
     mix release
 
-FROM alpine:3.16.2 as release
+FROM debian:11.6-slim as release
 
 RUN set -xe; \
-    apk add --update  --no-cache --virtual .runtime-deps \
+        apt-get update && \
+        TZ=Etc/UTC \
+        apt-get install -y \
         ca-certificates \
-        libmcrypt \
         libmcrypt-dev \
         openssl \
-        libstdc++ \
-        ncurses-libs \
+        libssl-dev\
+        wkhtmltopdf \
         tzdata;
 
 # Create a `bullhorn` group & user
 # I've been told before it's generally a good practice to reserve ids < 1000 for the system
 RUN set -xe; \
-    addgroup -g 1000 -S bullhorn; \
-    adduser -u 1000 -S -h /bullhorn -s /bin/sh -G bullhorn bullhorn;
+    addgroup --gid 1000 --system bullhorn; \
+    adduser --uid 1000 --system --home /bullhorn --shell /bin/sh --ingroup bullhorn bullhorn;
```

Comment:
You can get rid of the `addgroup` command and use:
```
adduser --uid 1000 --system --home /bullhorn --shell /bin/sh --group bullhorn;
```
The `--group` option creates a group with the same name and id as the user and automatically adds the user to that group.